modules/eap/pom.xml

```diff
@@ -65,6 +65,10 @@
             <artifactId>hibernate-ogm-couchdb</artifactId>
```

Comment:
The EAP module ZIP has been removed on master in parallel (it's severely outdated at this point and EAP 6 will not work with ORM 5). Could you thus exclude these changes from the PR? Thanks!